addons/web/static/tests/views/fields/text_field.test.js

```diff
@@ -0,0 +1,211 @@
+        type: "form",
+        resModel: "product",
+        resId: 1,
+        arch: `<form><field name="description" rows="40"/><field name="description"/></form>`,
+    });
+    const textareas = queryAll(".o_field_text textarea");
+    expect(textareas[0].rows).toBe(40);
+    expect(textareas[0].clientHeight).toBeGreaterThan(textareas[1].clientHeight);
+});
+
+test("is translatable", async () => {
+    Product._fields.description = fields.Text({ translate: true });
+    Product._records = [{ id: 1, description: "Description as text" }];
+    await makeMockServer({
+        multi_lang: true,
+    });
+    onRpc("/web/dataset/call_kw/res.lang/get_installed", () => {
+        return Promise.resolve([
+            ["en_US", "English"],
+            ["fr_BE", "French (Belgium)"],
+        ]);
+    });
+    onRpc("/web/dataset/call_kw/product/get_field_translations", () => {
+        return Promise.resolve([
+            [
+                { lang: "en_US", source: "Description as text", value: "Description as text" },
+                {
+                    lang: "fr_BE",
+                    source: "Description as text",
+                    value: "Description sous forme de texte",
+                },
+            ],
+            { translation_type: "text", translation_show_source: false },
+        ]);
+    });
+    await mountView({
+        type: "form",
+        resModel: "product",
+        resId: 1,
+        arch: `<form><sheet><group><field name="description"/></group></sheet></form>`,
+    });
+    expect(".o_field_text textarea").toHaveClass("o_field_translate");
+    expect(".o_field_text .btn.o_field_translate").toHaveCount(1);
+    await contains(".o_field_text .btn.o_field_translate").click();
+    expect(".modal").toHaveCount(1);
+});
+
+test("is translatable on new record", async () => {
+    Product._fields.description = fields.Text({ translate: true });
+    Product._records = [{ id: 1, description: "Description as text" }];
+    await makeMockServer({
+        multi_lang: true,
+    });
+    await mountView({
+        type: "form",
+        resModel: "product",
+        arch: `<form><sheet><group><field name="description"/></group></sheet></form>`,
+    });
+    expect(".o_field_text .btn.o_field_translate").toHaveCount(1);
+});
+
+test("press enter inside editable list", async () => {
+    Product._records = [{ id: 1, description: "Description as text" }];
+    await mountView({
+        type: "list",
+        resModel: "product",
+        arch: `
+            <list editable="top">
+                <field name="description" />
+            </list>`,
+    });
+    await contains(".o_data_row .o_data_cell").click();
+    expect("textarea.o_input").toHaveCount(1);
+    expect("textarea.o_input").toHaveValue("Description as text");
+    expect(queryOne("textarea.o_input")).toBe(document.activeElement);
+    expect("textarea.o_input").toHaveValue("Description as text");
+    await fieldTextArea("description").press("Enter");
```

Comment:
Note: press("Enter") doesn't add \n in inputs. 
JUM has been made aware and will fix it in the near future.
So we don't test that we added the \n for now.